manifests/h/Headlamp/Headlamp/0.25.1/Headlamp.Headlamp.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+PackageIdentifier: Headlamp.Headlamp
+PackageVersion: 0.25.1
+PackageLocale: en-US
+Publisher: Headlamp
```

```suggestion
Publisher: Kinvolk
```
This should match Control Panel.